tck/features/MatchAcceptance2.feature

```diff
@@ -578,31 +578,31 @@ Feature: MatchAcceptance2
     When executing query:
       """
       MATCH (a:A)-->(n)-->(m)
-      RETURN n.x, count(*)
-        ORDER BY n.x
+      RETURN n.num, count(*)
+        ORDER BY n.num
         LIMIT 1000
       """
     Then the result should be, in order:
-      | n.x | count(*) |
-      | 1   | 1        |
-      | 2   | 1        |
+      | n.num | count(*) |
+      | 1     | 1        |
+      | 2     | 1        |
     And no side effects
 
   Scenario: Simple node property predicate
     Given an empty graph
     And having executed:
       """
-      CREATE ({foo: 'bar'})
+      CREATE ({name: 'bar'})
       """
     When executing query:
       """
       MATCH (n)
-      WHERE n.foo = 'bar'
+      WHERE n.name = 'bar'
       RETURN n
       """
     Then the result should be:
-      | n              |
-      | ({foo: 'bar'}) |
+      | n                |
+      | ({name: 'bar'}) |
```

Comment:
looks like one space too many added in the first row

Reply:
Thanks for noticing. Fixed.